spec/spec_helper.rb

```diff
@@ -42,16 +43,6 @@
   config.expect_with(:rspec) { |c| c.syntax = [:should, :expect] }
 
   config.include FactoryGirl::Syntax::Methods
-
-  config.before do
-    [
-      BugzillaService,
-      GithubService,
-      MinigitService
-    ].each do |service_model|
-      allow_any_instance_of(service_model).to receive(:service).and_raise(
-        "Live execution is not allowed in specs.  Use stubs/expectations on service instead."
-      )
-    end
-  end
 end
+
+WebMock.disable_net_connect!(:allow_localhost => true)
```

Comment:
Oh woooops. Forgot about our special snowflake. Let's depend on this for HTTP services but I'll add minigit back for sure, as I don't think we really want to change that service interface anyway.